src/app/app.component.ts

```diff
@@ -1,4 +1,4 @@
-import { Component, NO_ERRORS_SCHEMA } from '@angular/core';
+import { Component, NO_ERRORS_SCHEMA, OnInit } from '@angular/core';
```

Comment:
`OnInit` is not used, please remove

```suggestion
import { Component, NO_ERRORS_SCHEMA } from '@angular/core';
```